README.md

```diff
@@ -96,6 +96,14 @@ or
 mybucket /path/to/mountpoint fuse.s3fs _netdev,allow_other 0 0
 ```
 
+When using fstab to mount a non-Amazon S3 implementation, the URL and
+path-style options also must be provided.
+
+```
+s3fs#mybucket /path/to/mountpoint fuse _netdev,allow_other,use_path_request_style,url=http://url.to.s3/ 0 0
+
```

Comment:
Extra space.

Reply:
Yeah definitely not needed, I just thought another example wouldn't hurt, especially for people who do not already understand how mount options are specified in fstab. 

I rarely find myself frustrated by excessive documentation, it's almost always the other way around. But it's your baby, if you don't want, that's cool.